Sources/Sentry/SentryDispatchFactory.m

```diff
@@ -11,13 +11,13 @@ - (SentryDispatchQueueWrapper *)queueWithName:(const char *)name
     return [[SentryDispatchQueueWrapper alloc] initWithName:name attributes:attributes];
 }
 
-- (SentryDispatchQueueWrapper *)createBackgroundQueueWithName:(const char *)name
-                                             relativePriority:(int)relativePriority
+- (SentryDispatchQueueWrapper *)createLowPriorityQueue:(const char *)name
+                                      relativePriority:(int)relativePriority
 {
     SENTRY_CASSERT(relativePriority <= 0 && relativePriority >= QOS_MIN_RELATIVE_PRIORITY,
         @"Relative priority must be between 0 and %d", QOS_MIN_RELATIVE_PRIORITY);
     dispatch_queue_attr_t attributes = dispatch_queue_attr_make_with_qos_class(
-        DISPATCH_QUEUE_SERIAL, QOS_CLASS_BACKGROUND, relativePriority);
+        DISPATCH_QUEUE_SERIAL, QOS_CLASS_UTILITY, relativePriority);
```

Comment:
`l`: small addition to explain why `QOS_CLASS_UTILITY` is considered a low priority. I can't suggest changes for unchanged lines, you'll have to add it manually.

```suggestion
    // The QOS_CLASS_UTILITY is defined in `qos.h` and used by the `DISPATCH_QUEUE_PRIORITY_LOW`, therefore it can be considered it a low priority queue
    dispatch_queue_attr_t attributes = dispatch_queue_attr_make_with_qos_class(
        DISPATCH_QUEUE_SERIAL, QOS_CLASS_UTILITY, relativePriority);
```

Reply:
We could also add a reference to the docs:
https://developer.apple.com/documentation/dispatch/dispatch_queue_priority_low#Discussion

Reply:
Good point @philprime 🙇 
Added the comment with 3e238f132b87c7c601af340cfb46dffc5efc0bba

Reply:
> We could also add a reference to the docs:
https://developer.apple.com/documentation/dispatch/dispatch_queue_priority_low#Discussion

Good idea 👍 
Sorry, I've missed the comment. Added the reference with cafb32f897d43c5a87d915d7128eb0b5a8c7589d

Reply:
No worries, this is good enough